install/linux/usr/bin/odemis-select-mic-start

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+: '
+This script is used when there are multiple modular configurations available for METEOR.
+It is recommended to combine it with the exec= command of the Odemis.desktop file.
+However it could be used stand-alone if it is required.
+The syntax to run this script -> odemis-select-mic-start [search_dir] [search_pat].
+The script will search for configuration (.yaml) files with the used search directory
+and pattern. Once files are found microscope names are extracted and placed in a list.
+This list is presented to the user in the form of a (zenity) selection dialog window.
+After selection of the microscope configuration the right yaml file will be used to start Odemis.
+'
+# extract the arguments which were used to start this script
+args=( "$@" )
+
+for arg in ${args[@]}; do
+    if test "$arg" == "--help"; then
+        # Display Help
+        echo
+        echo "Syntax: odemis-select-mic-start [search_dir] [search_pat]"
+        echo "arguments:"
+        echo "search_dir     The directory to search for finding the main configuration files."
+        echo "search_pat     The search pattern to use to filter out specific file names."
```

Comment:
I have some trouble to understand why you would need this, can you add a small example of a pattern?

Reply:
There needs to be a pattern to search for, specifically for finding the 'main' files. We don't want to any other yaml files as those would not contain a microscope name at all.
I have added a small example for searching METEOR files.